### PR TITLE
add page for using libraries without Amplify backend

### PR DESCRIPTION
**Pull request by hdworld11**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/start/libraries-standalone/index.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+Amplify client libraries provide you with the flexibility to directly connect your application to AWS resources, regardless of whether you choose to set up an Amplify backend environment or not. You can take advantage of the Amplify libraries while maintaining control over your AWS resources.
```
Comment (renebrandel) [inline src/pages/[platform]/start/libraries-standalone/index.mdx:33]:
```suggestion
Amplify client libraries provide you with the flexibility to directly connect your application to AWS resources, regardless of whether you choose to set up an Amplify backend environment or manually configure individual AWS services, such as AWS AppSync, Amazon Cognito, Amazon S3, and more. You can take advantage of the Amplify libraries while maintaining control over your AWS resources. 
```

File: src/pages/[platform]/start/libraries-standalone/index.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+import { Card } from '@aws-amplify/ui-react';
+
+export const meta = {
+  title: 'Connect to AWS resources',
```
Comment (josefaidt) [inline src/pages/[platform]/start/libraries-standalone/index.mdx:5]: can we align the title and slug?

File: src/pages/[platform]/start/libraries-standalone/index.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+Amplify client libraries provide you with the flexibility to directly connect your application to AWS resources, regardless of whether you choose to set up an Amplify backend environment or manually configure individual AWS services, such as AWS AppSync, Amazon Cognito, Amazon S3, and more. You can take advantage of the Amplify libraries while self-managing those resources outside of an Amplify backend. 
+
+If you prefer not to use the Amplify backend, you can still benefit from the Amplify libraries by following the instructions provided
```
Comment (renebrandel) [inline src/pages/[platform]/start/libraries-standalone/index.mdx:35]: I think this sentence is now redundant given the paragraph above.

File: src/pages/[platform]/start/libraries-standalone/index.mdx (hunk 2 above)
Comment (hdworld11) [reply src/pages/[platform]/start/libraries-standalone/index.mdx]: updated

File: src/pages/[platform]/start/connect-existing-aws-resources/index.mdx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+Amplify client libraries provide you with the flexibility to directly connect your application to AWS resources, regardless of whether you choose to set up an Amplify backend environment or manually configure individual AWS services, such as AWS AppSync, Amazon Cognito, Amazon S3, and more.
+
+If you prefer not to use the Amplify backend, you can still benefit from the Amplify libraries by following the instructions provided
```
Comment (renebrandel) [inline src/pages/[platform]/start/connect-existing-aws-resources/index.mdx:35]:
```suggestion
If you configured AWS resources outside of an Amplify backend, you can still use the Amplify libraries to connect to them by following the instructions provided:
```

File: src/pages/[platform]/start/connect-existing-aws-resources/index.mdx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+<Columns columns={2}>
+<Card variation="outlined">
+[Connect to Cognito](/[platform]/build-a-backend/auth/use-existing-cognito-resources/)
+
+Connect to Cognito resources using Amplify APIs for Auth
```
Comment (renebrandel) [inline src/pages/[platform]/start/connect-existing-aws-resources/index.mdx:41]:
```suggestion
Connect to Cognito resources using Amplify Auth's client library
```

**Review by renebrandel — APPROVED** [review]

**Review by josefaidt — APPROVED** [review]

**Review by timngyn — APPROVED** [review]

**Pull request merged**